pillar.example

```diff
@@ -3,23 +3,94 @@ postfix:
   config:
     smtpd_banner: $myhostname ESMTP $mail_name
     biff: 'no'
-
     append_dot_mydomain: 'no'
-
     readme_directory: 'no'
-
-    smtpd_tls_cert_file: /etc/ssl/certs/ssl-cert-snakeoil.pem
-    smtpd_tls_key_file: /etc/ssl/private/ssl-cert-snakeoil.key
-    smtpd_use_tls: 'yes'
-    smtpd_tls_session_cache_database: btree:${data_directory}/smtpd_scache
-    smtp_tls_session_cache_database: btree:${data_directory}/smtp_scache
-
     myhostname: localhost
-    alias_maps: hash:/etc/aliases
-    alias_database: hash:/etc/aliases
     mydestination: localhost, localhost.localdomain
     relayhost: 
     mynetworks: 127.0.0.0/8 [::ffff:127.0.0.0]/104 [::1]/128
     mailbox_size_limit: 0
     recipient_delimiter: +
     inet_interfaces: all
+    tls_random_source: dev:/dev/urandom
```

Comment:
I think the default source is /dev/random, which can block if the kernel is
running short of entropy. I found many discussions online where people set
the source device to urandom, which is a non blocking device. I don't think
urandom exists in all platforms but since this is an example file, I think
people will notice an error and will not take the line from pillar.example.
if you want, I can write a comment near the line to explain it better

On Mon, Jul 6, 2015, 13:16 Brian Jackson <notifications@github.com> wrote:

> In pillar.example
> <https://github.com/saltstack-formulas/postfix-formula/pull/19#discussion_r33958107>
> :
>
> >      mydestination: localhost, localhost.localdomain
> >      relayhost:
> >      mynetworks: 127.0.0.0/8 [::ffff:127.0.0.0]/104 [::1]/128
> >      mailbox_size_limit: 0
> >      recipient_delimiter: +
> >      inet_interfaces: all
> > +    tls_random_source: dev:/dev/urandom
>
> Did you make this change for a specific reason?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/saltstack-formulas/postfix-formula/pull/19/files#r33958107>
> .
>